main.py

```diff
@@ -8,7 +8,7 @@
 from typing import List, Union, Generator, Iterator
 
 
-from utils.pipelines.auth import bearer_security, get_current_user
+from utils.pipelines.auth import get_current_user_or_abort
```

Comment:
Should keep the `get_current_user` convention. Please refer to our main Open WebUI repo!

Reply:
Hi Timothy, I'm sorry but I don't understand what you mean. Are you talking about adding the HTTP exception in the `get_current_user` or implement something like the function `get_current_user_by_api_key` (https://github.com/open-webui/open-webui/blob/eff736acd2e0bbbdd0eeca4cc209b216a1f23b6a/backend/utils/utils.py#L116C5-L116C32)?